src/ghazal_azadi_ex/ex02/file_stats.py

```diff
@@ -0,0 +1,43 @@
+def char_counts(textfilename):
+    # Function which returns the number of occurrence of character code i.
+
+    import io
+    with io.open(textfilename, 'r', encoding='utf8') as file:
+        txt = file.read()
+    # Reading the file
+
+    order = []
+    for i in txt:
+        order.append(ord(i))
+    # Unicode code point of the character
+
+    srt = {}
+    for i in order:
+        srt[i] = order.count(i)
+    # Counting the unicode
+
+    result = [0] * 256
+    index = list(srt.keys())
+    value = list(srt.values())
```

Comment:
You may want to check the suggested solution for a easier way to do this exercise